html/webappapis/structured-clone/structured-clone-battery-of-tests.js

```diff
@@ -592,3 +592,17 @@ structuredCloneBatteryOfTests.push({
   },
   requiresDocument: true
 });
+
+check('ObjectPrototype must lose its exotic-ness when cloned',
+  () => Object.prototype,
+  (copy, original) => {
+    assert_not_equals(copy, original);
+    assert_true(copy instanceof Object);
+
+    const newProto = { some: 'proto' };
+    // Must not throw:
+    Object.setPrototypeOf(copy, newProto);
```

Comment:
How likely is it that this affects the test framework somehow?

Reply:
I can't see any way it would. The possible things that I could imagine happening are:

- The browser prevents cloning and this line is never reached. (Safari, current spec)
- The browser creates a new non-exotic object and this just sets the prototype of that new object. (Firefox, Chrome, proposed spec)
- The browser implements extra logic that makes the clone of %ObjectPrototype% be the other realm's %ObjectPrototype%, or some new immutable prototype exotic object, and this throws with no impact. (Nobody does that.)